### ISSUE TITLE
Logging doesn’t support renaming of values

### ISSUE THREAD

**Issue by kstephan-wescale** [GoogleCloudPlatform/ops-agent#246]
When parsing input data like `{"severity": "warn"}` the value can’t be renamed to “WARNING” for GCP Logging to properly handle it.  As in #245 I propose to implement [fluentbit’s Modify filter](https://docs.fluentbit.io/manual/pipeline/filters/modify) and add a `filters` element to the configuration model, with that it’s as easy as:
```
condition key_value_equals severity warn
set severity WARNING
```

**Comment by qingling128** [comment]
This is WIP.

**Comment by quentinmit** [comment]
Yeah, this is PR #474

**Comment by braydonk** [comment]
This is now implemented in the Ops Agent as of 2.15.0.
